angular/projects/admin-nrpti/src/app/mines/mines-records-detail/mines-records-detail.component.html

```diff
@@ -0,0 +1,82 @@
+<main class="container-fluid-padding">
```

Comment:
Seems to be missing the "date" attribute display as per the AC? Not sure which "date" is supposed to be here though... kinda unclear in the ticket.

Reply:
It actually shows the date on line 14. If there is an updated date it will say "Updated on blah" if it is missing then it will use the created date and say "Created on blah."  This follows how the date is displayed in other areas, but if it isn't obvious enough I could change it into a field like type/agency.

Reply:
No that's probably fine. The design has it as a defined field... but for a create/update date a user should not be able to change it so having it the way you've done makes sense.